frontend/sonar-project.properties

```diff
@@ -6,15 +6,14 @@ sonar.projectName=PIMS Frontend (React)
 
 # Paths below are relative to the sonar-project.properties file. Defaults to .
 sonar.sources=src
-sonar.exclusions=**/node_modules/**, **/coverage/**, **.spec.**
+sonar.exclusions=node_modules/**, coverage/**, src/**/*.spec.**, src/**/*.test.**
 
 sonar.tests=src
-sonar.test.inclusions=**.spec.**
+sonar.test.inclusions=src/**/*.spec.**, src/**/*.test.**
 
-# --- TODO: integrate code coverage reports into SonarQube ---
-
-# sonar.javascript.lcov.reportPaths=frontend/coverage/lcov.info
-# sonar.testExecutionReportPaths=frontend/coverage/test-report.xml
+# --- integrate code coverage reports into SonarQube ---
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+sonar.testExecutionReportPaths=test-report.xml
```

Comment:
- `lcov.info` is generated by Jest
- `test-report.xml` is generated by `jest-sonar-reporter`